implementations/rust/ockam/ockam_node/src/node.rs

```diff
@@ -30,18 +28,26 @@ pub fn start_node() -> (Context, Executor) {
     (ctx, exe)
 }
 
-/// Utility to setup tracing-subscriber from the environment
+/// Utility to setup tracing-subscriber from the environment.
+///
+/// Does nothing if the `no_init_tracing` feature is enabled (for now -- this
+/// should be improved, though).
 fn setup_tracing() {
     #[cfg(feature = "std")]
-    {
-        let filter = EnvFilter::try_from_env("OCKAM_LOG").unwrap_or_else(|_| {
-            EnvFilter::default()
-                .add_directive(LevelFilter::INFO.into())
-                .add_directive("ockam_node=info".parse().unwrap())
+    if !cfg!(feature = "no_init_tracing") {
```

Comment:
Could you please recheck if logging is still working after this change? (It wasn't during your demo)

Reply:
Hm, yes, this seems broken still, I'll figure out why.